### ISSUE TITLE
Flight Data Record Fails - GR6 & GR8

### ISSUE THREAD

**Issue by Smoothkd** [FrSkyRC/ETHOS-Feedback-Community#4664]
### Flight Data Record (FDR) Fails to function.  Tested on Archer Plus GR6 and GR8.  

- Transmitter: X18S 
- Ethos version 1.15.15
- TD-ISRM: HW 1.0.1 FW: 2.2.7 (FCC)
- Archer Plus GR6 Firmware: 1.01.10 
- Archer Plus GR8

**GR6 1.05 Firmware Symptoms:** 
With Rx registered and bound. When FDR option is chosen for RX1, Dialogue box shows "Waiting for Rx" after short period new dialogue box shows "No Response from RX".   No options but to exit. 

**GR6 1.01.10 Firmware Symptoms:**  
Updated firmware OTA to 1.01.10 successful.  When FDR option is chosen for RX1, It Unbinds and Unregisters Rx. Solid Green light goes to Red. In order to to re-register and bind, I had to change Owner registration ID from default to new set of characters!!? 

**GR8 1.05 Firmware Symptoms:** 
Same as GR6 with FW 1.05

Screenshots: 
![FDR](https://github.com/user-attachments/assets/2a39d1a4-b32b-4255-9e84-0eb1cb03f7c5)
![FDR1](https://github.com/user-attachments/assets/d2f65141-f546-4be3-b257-e4310b9ce1db)
![FDR2](https://github.com/user-attachments/assets/48774194-eec5-48c4-9ca6-b507bbca40fc)

**Comment by mawzthefinn** [comment]
This is a known receiver limitation. The functionality has not been enabled in receiver firmware. Please open an issue here:

https://github.com/FrSkyRC/RF_Feedback_Group/issues?page=2&q=is%3Aissue+is%3Aopen

**Comment by Smoothkd** [comment]
Wow!  Couple of Questions please: 

1- This functionality is clearly listed on FrSky site as a feature almost on all Archer Plus, TW and some of the TD receivers.  None have this feature enabled? 
2- What needs to be done to ensure the GR6 updated to firmware 1.01.10 is reliable to fly.  Especially after accessing the FDR option crashes the receiver.  Should it be reset to factory defaults / downgraded to FW 1.05 Or nothing needs to be done? 

Wish I could at least see the utility of this "feature".   I'll open an issue in RF_Feedback_Group with same content as I have here.  Unless you advise a different format so it gets prioritised. Especially since this is a known issue.

**Comment by mawzthefinn** [comment]
1. Yes, it's supposed to be there. It is there on the TD receivers, but some of the TW and AP series are missing it in the current firmware.

2. The receiver is safe to fly. The issue is just that the receiver doesn't respond with Blackbox data when you access the FDR. 

The info you included here is the necessary info. I definitely recommend posting the same thing in the RF Feedback Group as that is the correct spot for a receiver firmware issue like this one.

**Comment by Smoothkd** [comment]
Done. Opened issue on RC_Feedback.  Hope it'll get addressed.  I reported this to support almost 2 months ago. 

My only concern is having updated to FW v1.01.10 and then accessing FDR crashed the GR6.  I had to change owner ID to Reg and Bind again. Don't understand Owner ID role in this equation.   Not sure if it broke something else on the unit.  This is on a relatively expensive and hard to get racing glider.  Just want to make sure I am covering all the basis before I got with it.  I'm converting from JR to FrSky. 

Many thanks.
